### DISCUSSION TITLE
Uncaught SyntaxError: Unexpected end of JSON input

### DISCUSSION THREAD

**Discussion by lislmaka** [livewire/livewire#3301] [7. Bugs (archived)] [answered]
Hi, 

laravel: "8.49.2"
livewire: "2.5.3"
vue: "2.6.14"

livewire / vue do not work on the same page

**UPD - livewire: "2.5.1" works good**

![error](https://user-images.githubusercontent.com/70829116/125223384-5c097580-e2d4-11eb-948a-4a8927b0db4c.jpg)

**Comment by joshhanley** [comment, marked as answer]
@lislmaka a fix has been tagged for this, please upgrade to Livewire 2.5.5 and let us know if you are still having any issues 🙂

**Reply by lislmaka** [reply]
@joshhanley 
Everything is working. Thank you very much for your promptness
